test/api/expiretask_test.js

```diff
@@ -63,4 +63,33 @@ suite('Task Expiration (expire-tasks)', function() {
       assume(err.statusCode).equals(404);
     });
   });
+
+  test("expire won't drop table", async () => {
```

Comment:
The task table...

The expires test we had, only tested that things got deleted... You can delete things by dropping the entire table, and our test would still pass... So this is a test to prove that expiration of tasks doesn't drop the entire table. He he :)